hypervisor/src/kvm/mod.rs

```diff
@@ -901,6 +901,11 @@ impl vm::Vm for KvmVm {
     fn as_any(&self) -> &dyn Any {
         self
     }
+
+    #[cfg(feature = "sev_snp")]
```

Comment:
Is there a need to define this for KVM, since we don't support sev_snp on KVM?

Reply:
Yes. If we want to enable both kvm and mshv feature during build. It does not do anything though